py_mind_memo/graphics.py

```
@@ -230,7 +234,11 @@ def get_font_metrics(style):
             max_w = max(max_w, line_w)
             total_h += (line_max_h if line_max_h > 0 else size + 10)
             
-        result = max(100, max(max_w + 20, img_w + 20)), max(35, total_h + 12 + img_h)
+        if node.image_data and node.id in self.image_cache and self.image_cache[node.id][0].width() == 20 and self.image_cache[node.id][0].height() == 20:
+            result = max(100, max_w + img_w + 20), max(35, total_h + 12)
+        else:
+            result = max(100, max(max_w + 20, img_w + 20)), max(35, total_h + 12 + img_h)
```

Ruff (0.15.6)

[warning] Nested `max` calls can be flattened

Flatten nested `max` calls

(PLW3301)